test/log-generators/pipelines/install.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+# Create a CronJob that run every minute, submitting a Tekton pipeline that generates an Apache
+# log file 1024 lines in length.
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

Comment:
What is the `SCRIPT_DIR` variable used for? I don't see it used anywhere in the script. If it is used in something called by this code, could you please add a comment with that information?

Reply:
It's there because I expect @maruiz93 to request other changes that will need it!

Reply:
If you end up not needing it, you should remove it. See also, [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it).

Reply:
YAGNI!  Love it. :-)